src/cag/context_builder.py

```
@@ -0,0 +1,896 @@
+
+    def _build_all_contexts(
+        self,
+        context: CAGContext,
+        data: Dict[str, Any],
+        zip_codes: List[str]
+    ) -> CAGContext:
+        """Build all context types."""
+        for context_type, builder in self._context_builders.items():
+            try:
+                builder(context, zip_codes, data)
+            except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)